webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'production',
+  entry: {
+    index: './src/index.js',
+    data: './src/data.js',
```

Comment:
- [ ] [OPTIONAL] your entry point is `index` so you don't need this extra key. you only need the `index` attribute in this section